### PR TITLE
RavenDB-18122 Cluster View: Provide info about how to bootstrap the c…

### PR DESCRIPTION
**Pull request by Danielle9897**
…luster

### Issue link
https://issues.hibernatingrhinos.com/issue/RavenDB-18122

### Additional description
Added text info & hide URL when passive

### Type of change
- Bug fix

### How risky is the change?
- Low 

### Backward compatibility
- Non breaking change

### Is it platform specific issue?
- No

### Documentation update
- No documentation update is needed 

### Testing 
- It has been verified by manual testing

### Is there any existing behavior change of other features due to this change?
- No

### UI work
- No UI work is needed

### CODE REVIEW

File: src/Raven.Studio/wwwroot/App/views/manage/cluster.html [hunk 1]
```diff
@@ -27,6 +27,20 @@
...
             </div>
+            <div data-bind="visible: topology().isPassive()">
+                <div class="passive-node-info bg-info margin-top">
+                    <div class="padding">
+                        <p class="lead margin-bottom">The running server is in a <strong>Passive State</strong>, it is not part of a cluster yet.</p>
+                        <p>Either one of the following can be done to <strong>Bootstrap a Cluster</strong>:</p>
+                        <ul>
+                            <li>Create a new database</li>
+                            <li>Bootstrap the cluster (see button above)</li>
+                            <li>Add another node (see button above), resulting in both nodes being part of the cluster</li>
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/App/views/manage/cluster.html:38]: another means there is already some node and we add second one?

File: src/Raven.Studio/wwwroot/Content/css/styles-common.less [hunk 2]
```diff
@@ -2058,6 +2058,16 @@ ul.properties {
     border-color: @brand-warning !important;
 }
 
+.passive-node-info {
+    width: 100%;
+    padding: @gutter;
+    margin-bottom: @gutter !important;
+    background-image: url(../img/pages/about/passive-state-bg.svg);
```
Comment (ml054) [inline src/Raven.Studio/wwwroot/Content/css/styles-common.less:2065]: are you sure this is correct url? 
You moved that from `css/pages/about.less` to `css/styles-common.less` so I think we need another `../` in url, right?

File: src/Raven.Studio/wwwroot/App/views/manage/cluster.html (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/cluster.html:38]: yes, there is the existing (Passive) node, and you may add another one, 
which will result in both nodes being in a new cluster.
I'm checking that now again.

Comment (ml054) [reply src/Raven.Studio/wwwroot/App/views/manage/cluster.html:38]: if you already checked that then no need to test that again.

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/cluster.html:38]: so theoretically, it should work,
but when I try to add another (passive) node to the current passive node
I am always required to register a license first... 
which is an action that by itself causes the first node to become a cluster..
so adding the node is not a relevant action anymore

Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/App/views/manage/cluster.html:38]: As discussed w/ @grisha-kotler 
yes - adding another node will result in both of them being in a cluster
so - leaving this text as-is.

File: src/Raven.Studio/wwwroot/Content/css/styles-common.less (hunk 2 above)
Comment (Danielle9897) [reply src/Raven.Studio/wwwroot/Content/css/styles-common.less:2065]: As discussed now with @kwiato 
the path is not relative to the LESS file location,
it is relative to the CSS file generated from the LESS
(and it is working)

Comment (ml054) [reply src/Raven.Studio/wwwroot/Content/css/styles-common.less:2065]: When doing review I don't checkout your or anyone else code and sometimes I do blind guesses. If you know something is working fine - just say so and we move forward. :) No need for proves, etc. :)

**Review by ml054 — APPROVED** [review]

**Pull request merged**